apps/web/src/services/asset-inventory/collector/collector-main/modules/collector-item-info/CollectorItemJobList.vue

```diff
@@ -2,84 +2,91 @@
     <div class="info-item">
         <p class="info-label">
             {{ $t('INVENTORY.COLLECTOR.MAIN.RECENT_JOBS') }}
+            <p-tooltip
+                :contents="$t('INVENTORY.COLLECTOR.MAIN.RECENT_JOBS_TOOLTIP')"
+                position="top-end"
+            >
+                <p-i name="ic_question-mark-circle"
+                     class="title-tooltip"
+                     height="0.875rem"
+                     width="0.875rem"
+                     color="inherit"
+                />
+            </p-tooltip>
         </p>
         <div class="jobs-wrapper">
             <div v-for="(jobStatus, index) in props.item.recentJobAnalyze"
                  :key="`job-item-${index}`"
                  class="jobs-contents"
                  @click.stop
             >
-                <p-tooltip v-if="jobStatus.status === JOB_STATE.SUCCESS"
-                           class="icon-fill-wrapper success"
-                           :contents="$t('INVENTORY.COLLECTOR.MAIN.JOB_SUCCESS', {date: dayjs.utc(jobStatus.finished_at).tz(storeState.timezone).format('YYYY-MM-DD hh:mm:ss')})"
-                           position="top"
+                <router-link :to="{ name: ASSET_INVENTORY_ROUTE.COLLECTOR.HISTORY.JOB._NAME, params: { jobId: jobStatus.job_id} }"
+                             class="icon-fill-wrapper"
                 >
-                    <router-link :to="{ name: ASSET_INVENTORY_ROUTE.COLLECTOR.HISTORY.JOB._NAME, params: { jobId: jobStatus.job_id} }">
+                    <p-tooltip v-if="jobStatus.status === JOB_STATE.SUCCESS"
+                               :contents="$t('INVENTORY.COLLECTOR.MAIN.JOB_SUCCESS', {date: dayjs.utc(jobStatus.finished_at).tz(storeState.timezone).format('YYYY-MM-DD hh:mm:ss')})"
+                               position="top-end"
+                               class="job-tooltip"
+                    >
```

Comment:
nitpick: It seems like there are 3 very similar parts that are repeated. It should be fine to use a for loop!